### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by i-s-h-i-k-a**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Checklist

- [ ] Tests have been added or updated to cover the changes
- [ ] Documentation has been updated to reflect the changes
- [ ] Code follows the established coding style guidelines
- [ ] All tests are passing

### CODE REVIEW

**Comment by nishant0708** [comment]
@i-s-h-i-k-a 
image
yeh forget password page mai changes nhi aayenge

File: src/ForgetPassword/Forget.jsx
Comment (nishant0708) [inline src/ForgetPassword/Forget.jsx:1]: Remove this Changes

**Review by nishant0708 — CHANGES_REQUESTED** [review]

**Comment by i-s-h-i-k-a** [comment]
@nishant working fine on my system

**Comment by Kartikey2011yadav** [comment]
🤣🤣🤣🤣🤣🤣

**Comment by Kartikey2011yadav** [comment]
Ye kya chal raha hai

**Comment by nishant** [comment]
Please @ the full username. I have no affiliation with this repository

**Review by nishant0708 — APPROVED** [review]

**Pull request merged**